packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -279,6 +283,21 @@ export class AccountResolver {
       .orderBy('createdAt', 'DESC');
   }
 
+  @Features(AdminPanelFeature.AccountSearch)
+  @ResolveField()
+  public async accountEvents(@Root() account: Account) {
+    // Not sure the best way for admin panel to get this config from event broker config
+    const eventsDbRef = this.firestore.collection('fxa-eb-users');
+    const queryResult = await eventsDbRef
+      .doc(account.uid)
```

Comment:
Seems like this could create a very large list. I think we should apply some sort of upper limit for now. If the limit is exceeded, the interface should indicate this, and we maybe we need to file an issue to support pagining. I suppose we can deal with this if/when it happens.